docs/docs/spark-configuration.md

```diff
@@ -155,16 +155,18 @@ spark.read
+| as-of-timestamp                     | (latest)              | A timestamp in milliseconds; the snapshot used will be the snapshot current at this time.                                                                                                                                                                 |
+| split-size                          | As per table property | Overrides this table's read.split.target-size and read.split.metadata-target-size                                                                                                                                                                         |
+| lookback                            | As per table property | Overrides this table's read.split.planning-lookback                                                                                                                                                                                                       |
+| file-open-cost                      | As per table property | Overrides this table's read.split.open-file-cost                                                                                                                                                                                                          |
+| vectorization-enabled               | As per table property | Overrides this table's read.parquet.vectorization.enabled                                                                                                                                                                                                 |
+| batch-size                          | As per table property | Overrides this table's read.parquet.vectorization.batch-size                                                                                                                                                                                              |
+| stream-from-timestamp               | (none) | A timestamp in milliseconds to stream from; if before the oldest known ancestor snapshot, the oldest will be used                                                                                                                                         |
+| streaming-max-files-per-micro-batch | INT_MAX | Maximum number of files per microbatch                                                                                                                                                                                                                    | 
+| streaming-max-rows-per-micro-batch  | INT_MAX | Maximum number of rows per microbatch. Note : smallest granuality supported is 1 file, please make sure number of records per file is always greater than the number of records in the largest file possible, otherwise it can lead to stream being stuck | 
```

Comment:
For the Description, I suggest "Maximum number of rows per microbatch. This number should be greater than the number of records in any data file in the table. The smallest unit that will be streamed is a single file, so if a data file contains more records than this limit, the stream will get stuck at this file."

(This avoids using the word "granularity", which is misspelled here as "granuality".)

Reply:
Just curious - what is our usual practice for updating a table? Do we reformat all the rows to align the column widths (which is only partially done here)? Or should we minimize churn and not reformat existing rows?

Reply:
sounds fair, changed the wording 

>Do we reformat all the rows to align the column widths (which is only partially done here)? Or should we minimize churn and not reformat existing rows 

Honestly, I am not sure of any practice, IDE changed the indentation for me, i fixed it back.